fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -823,14 +832,37 @@ ACTOR Future<Void> rocksDBMetricLogger(UID id,
 			cum = stat;
 		}
 
-		for (auto& p : propertyStats) {
+		for (auto& p : intPropertyStats) {
 			auto& [name, property] = p;
 			stat = 0;
 			// GetAggregatedIntProperty gets the aggregated int property from all column families.
 			ASSERT(db->GetAggregatedIntProperty(property, &stat));
 			e.detail(name, stat);
 		}
 
+		std::string propValue;
+		for (auto& p : strPropertyStats) {
+			auto& [name, property] = p;
```

Comment:
Can be simplified as:
` for (const auto& [name, property] : strPropertyStats) {`